Math A7800: Homework 6

Name: Jorge Monzon Diaz

Email: jorgemd503@gmail.com

## Dependencies

In [1]:
import numpy as np
from IPython.display import Math
import pandas as pd
import scipy.stats as stats

\pagebreak
# Exercise 5.15

Let $X_{ji}$ and $X_{jk}$ be the ith and kth components, respectively of $\pmb{X}_j$

## (a) Show that $\mu_i=E(X_{ji})=p_i$ and $\sigma_{ii}=Var(X_{ji})=p_i(1-p_i), i=1,2,...,p$

Since we are given discrete random variables

$$\mu=E(X_{ji})=1(p_i)+0(1-p_i)=p_i$$

$$\sigma_{ii}=Var(X_{ji})=(p_i)(1-\mu)^2+(1-p_i)(0-\mu)^2=(p_i)(1-pi)^2+(1-p_i)(pi)^2$$

$$=p_i(1-p_i)(1-p_i+p_i)=p_i(1-p_i)$$

## (b) Show that $\sigma_{ik}=Cov(X_{ji},X_{jk})=-p_ip_k, i\neq k$. Why must this covariance be necessarily negative?

Due to independence

$$\sigma_{ik}=Cov(X_{ji},X_{jk})=E(X_{ji},X_{jk})-E(X_i)E(X_j)=0-p_ip_k=-p_ip_k$$

This covariance is then always negative as there is no correlation between the variables.

\pagebreak

# Exercise 5.16

Given:

| $\hat p_1$ | $\hat p_2$ | $\hat p_3$ | $\hat p_4$ | $\hat p_5$ |
|--------|--------|--------|--------|----------------------------|
| .30    | .33    | .16    | .07    | .14                        |

$p_5=1-(p_1+p_2+p_3+p_4)$

## (a) Construct simultaneous 95% confidence intervals for $p_1,p_2,...,p_5$.

$$\pmb{\hat p}\pm\sqrt{\chi_q^2(\alpha)}\sqrt\frac{\pmb{\hat\Sigma}}{n}$$

Since we have 5 categories, q=4. We also know n=355.

$$\hat p_i\pm\sqrt{\chi_4^2(0.05)}\sqrt\frac{{\hat p_i(1-\hat p_i)}}{355}$$

In [2]:
p=[.30,.33,.16,.07,.14]

for i in range(0,5):
    #display CI for each p
    j=i+1
    display(Math('\\text{CI for }p_{%i}' %j))
    display(Math('%.2f\pm\sqrt{\chi_4^2(0.05)}\sqrt{\\frac {%.2f(1-%.2f)}{355}}' %(p[i],p[i],p[i])))
    
    #compute CI bounds and display them
    lower=p[i]-np.sqrt(stats.chi2.ppf(1-.05,4))*np.sqrt((p[i]*(1-p[i]))/355)
    upper=p[i]+np.sqrt(stats.chi2.ppf(1-.05,4))*np.sqrt((p[i]*(1-p[i]))/355)
    display(Math('(%.4f, %.4f)' %(lower,upper)))
    print("\n")

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

## (b) Construct a simultaneous 95% confidence interval that allows a comparison of the Bank of Shorewood with its major competitor, Bank B. Interpret this interval.

CI for difference in proportions:

$$\hat p_i - \hat p_j\pm\sqrt{\chi_q^2(\alpha)}\sqrt\frac{\hat p_i(1-\hat p_i)+\hat p_j (1-\hat p_j) - 2 \hat p_i \hat p_j}{n}$$

Bank of Shorewood: $p_1$

Bank B: $p_2$

$$\hat p_1 - \hat p_2\pm\sqrt{\chi_4^2(0.05)}\sqrt\frac{\hat p_1(1-\hat p_1)+\hat p_2 (1-\hat p_2) - 2 \hat p_1 \hat p_2}{355}$$

In [3]:
#display CI formula
display(Math('%.2f - %.2f\pm\sqrt{\chi_4^2(0.05)}\sqrt\\frac{%.2f(1-%.2f)+%.2f (1-%.2f) - 2 (%.2f)( %.2f)}{355}' %(p[0],p[1],p[0],p[0],p[1],p[1],p[0],p[1])))

#compute CI bounds and display them
lower=p[0]-p[1]-np.sqrt(stats.chi2.ppf(1-.05,4))*np.sqrt(((p[0]*(1-p[0]))+(p[1]*(1-p[1]))-2*p[0]*p[1])/355)
upper=p[0]-p[1]+np.sqrt(stats.chi2.ppf(1-.05,4))*np.sqrt(((p[0]*(1-p[0]))+(p[1]*(1-p[1]))-2*p[0]*p[1])/355)
display(Math('(%.4f, %.4f)' %(lower,upper)))

<IPython.core.display.Math object>

<IPython.core.display.Math object>

Since the simultaneous confidence is so small, we can claim there is no significant difference between the two proportions.

\pagebreak

# Exercise 5.17

Given:

| $\hat p_1$ | $\hat p_2$ | $\hat p_3$ |
|--------|--------|--------|
| .585    | .310    | .105    |

Construct the 95% simultaneous confidence intervals for the three proportions $p_1, p_2, \text{and } p_3=1-(p_1+p_2)$

$$\pmb{\hat p}\pm\sqrt{\chi_q^2(\alpha)}\sqrt\frac{\pmb{\hat\Sigma}}{n}$$

Since we have 3 categories, q=2. We also know n=200.

$$\hat p_i\pm\sqrt{\chi_2^2(0.05)}\sqrt\frac{{\hat p_i(1-\hat p_i)}}{200}$$

In [4]:
p=[.585,.310,.105]

for i in range(0,3):
    #display CI for each p
    j=i+1
    display(Math('\\text{CI for }p_{%i}' %j))
    display(Math('%.2f\pm\sqrt{\chi_2^2(0.05)}\sqrt{\\frac {%.2f(1-%.2f)}{355}}' %(p[i],p[i],p[i])))
    
    #compute CI bounds and display them
    lower=p[i]-np.sqrt(stats.chi2.ppf(1-.05,2))*np.sqrt((p[i]*(1-p[i]))/200)
    upper=p[i]+np.sqrt(stats.chi2.ppf(1-.05,2))*np.sqrt((p[i]*(1-p[i]))/200)
    display(Math('(%.4f, %.4f)' %(lower,upper)))
    print("\n")

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

\pagebreak

# Exercise 5.30

Refer to the data on energy consumption in Exercise 3.18

$x_1 =$ petroleum

$x_2 =$ natural gas

$x_3 =$ hydroelectric power

$x_4 =$ nuclear electric power

$$\pmb{\bar{x}}=\begin{bmatrix}0.766\\0.508\\0.438\\0.161\end{bmatrix}$$

$$ \pmb{S}= \begin{bmatrix}0.856&0.635&0.173&0.096\\0.635&0.568&0.128&0.067\\0.173&0.127&0.171&0.039\\0.096&0.067&0.039&0.043\end{bmatrix} $$

## (a) Obtain the large sample 95% Bonferroni confidence intervals for the mean consumption of each of the four types, the total of the four, and the difference, petroleum minus natural gas

Bonferroni CI:

$$\bar{x_i}-t_{n-1}(\frac{\alpha}{2m})\sqrt{\frac{s_{ii}}{n}}\leq\mu_i\leq\bar{x_i}+t_{n-1}(\frac{\alpha}{2m})\sqrt{\frac{s_{ii}}{n}}$$

$$\left[\bar{x_i}-t_{n-1}(\frac{\alpha}{2m})\sqrt{\frac{s_{ii}}{n}},\bar{x_i}+t_{n-1}(\frac{\alpha}{2m})\sqrt{\frac{s_{ii}}{n}}\right]$$


For large samples

$$\left[\bar{x_i}-z(\frac{\alpha}{2m})\sqrt{\frac{s_{ii}}{n}},\bar{x_i}+z(\frac{\alpha}{2m})\sqrt{\frac{s_{ii}}{n}}\right]$$

m=6

For $\mu_1,\mu_2,\mu_3,\mu_4$, their respective corresponding variance is $s_{11},s_{22},s_{33},s_{44}$

Let $\mu_{tot}$ be the mean for the total, and $s_{tot}$ be its corresponding variance.

$$\overline{x_{tot}}=\sum_{i=1}^4(\bar{x_i})=1.873$$

$$s_{tot}=Var(x_1+x_2+x_3+x_4)=\sum_{i=1}^4\sum_{j=1}^4(s_{i,j})=3.913$$

Let $\mu_{1-2}$ be the mean for petroleum minus natural gas, and $s_{1-2}$ be its corresponding variance.

$$\overline{x_{1-2}}=\bar{x_1}-\bar{x_2}=0.258$$

$$s_{1-2}=Var(x_1-x_2)=Var(x_1)+Var(x_2)-2Cov(x_1,x_2)=s_{11}+s_{22}-2s_{12}=0.154$$

In [5]:
labels=["petroleum","natural gas", "hydroelectric power", "nuclear electric power", "total", "petroleum minus natural gas"]
xbar=[0.766,0.508,0.438,0.161,1.873,0.258]
s=[0.856,0.568,0.171,0.043,3.913,0.154]


for i in range(len(labels)):
    #print equations
    display(Math('\\text{Large sample Bonferroni CI for %s}' %labels[i]))
    display(Math('\left[%.3f-z(\\frac{0.05}{12})\sqrt{\\frac{%.3f}{50}},%.3f+z(\\frac{0.05}{12})\sqrt{\\frac{%.3f}{50}}\\right]' %(xbar[i],s[i],xbar[i],s[i])))
    
    #compute CI bounds and display them
    lower=xbar[i]-stats.norm.ppf(1-0.05/12)*np.sqrt(s[i]/50)
    upper=xbar[i]+stats.norm.ppf(1-0.05/12)*np.sqrt(s[i]/50)
    display(Math('(%.3f, %.3f)' %(lower,upper)))
    print("\n")

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

## (b) Obtain the large sample 95% simultaneous $T^2$ intervals for the means consumption of each of the four types, the total of the four, and the difference, petroleum minus natural gas. Compare with your results for Part a.

For large samples

$$\bar{x_i}-\sqrt{\chi_p^2(\alpha)}\sqrt{\frac{s_{ii}}{n}}\leq\mu_i\leq\bar{x_i}+\sqrt{\chi_p^2(\alpha)}\sqrt{\frac{s_{ii}}{n}}$$

$$\left[\bar{x_i}-\sqrt{\chi_p^2(\alpha)}\sqrt{\frac{s_{ii}}{n}},\bar{x_i}+\sqrt{\chi_p^2(\alpha)}\sqrt{\frac{s_{ii}}{n}}\right]$$

We already computed the remaining means and sample variances in the previous problem so now we can just plug into our equation and compute

In [6]:
for i in range(len(labels)):
    #print equations
    display(Math('\\text{Large simultaneous }T^2\\text{-interval for %s}' %labels[i]))
    display(Math('\left[%.3f-\sqrt{\chi_4^2(0.05)}\sqrt{\\frac{%.3f}{50}},%.3f+\sqrt{\chi_4^2(0.05)}\sqrt{\\frac{%.3f}{50}}\\right]' %(xbar[i],s[i],xbar[i],s[i])))
    
    #compute CI bounds and display them
    lower=xbar[i]-np.sqrt(stats.chi2.ppf(1-0.05,4))*np.sqrt(s[i]/50)
    upper=xbar[i]+np.sqrt(stats.chi2.ppf(1-0.05,4))*np.sqrt(s[i]/50)
    display(Math('(%.3f, %.3f)' %(lower,upper)))
    print("\n")

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

<IPython.core.display.Math object>

\pagebreak

# Exercise 5.18a

Use the college test data in Table 5.2

Test the null hypothesis $H_0:\pmb{\mu}^T=[500,50,30]$ versus $H_1:\pmb{\mu}^T\neq[500,50,30]$ at the $\alpha=.05$ level of significance. Suppose $[500,50,30]^T$ represent average scores for thousands of college students over the last 10 years. Is there reason to believe that the group of students represented by the scores in Table 5.2 is scoring differently? Explain.

$$T^2=n(\pmb{\bar{X}}-\pmb{\mu_0})^T \pmb{S^{-1}}(\pmb{\bar{X}}-\pmb{\mu_0})$$

$$S=\frac{1}{n-1}(\pmb{X-1\bar{x}^T})^T(\pmb{X-1\bar{x}^T})$$

In [7]:
#read data
x = pd.read_excel (r'hw6.xlsx', sheet_name='data 518')

#compute mean
xmean=x.mean(axis = 0) 
print("x-mean:")
print(xmean,"\n")

#strip values from table
X=x.to_numpy()

#find S
R=X-np.ones((87,1))*[xmean[0],xmean[1],xmean[2]]
S=1/86*(np.matmul(R.transpose(),R))
print('S:')
print(S)

#invert S
invS=np.linalg.inv(S)
print('\nS^(-1):')
print(invS)

#compute T^2
T2=87*np.matmul(np.matmul(np.array([xmean[0]-500,xmean[1]-50,xmean[2]-30]),invS),np.array([xmean[0]-500,xmean[1]-50,xmean[2]-30]).transpose())
print('\nT^2:')
print(T2)

x-mean:
x1    526.586207
x2     54.689655
x3     25.126437
dtype: float64 

S:
[[5808.05934242  597.83520449  222.02967121]
 [ 597.83520449  126.05372895   23.38853248]
 [ 222.02967121   23.38853248   23.11173483]]

S^(-1):
[[ 0.00043194 -0.00157423 -0.0025565 ]
 [-0.00157423  0.0155044  -0.00056675]
 [-0.0025565  -0.00056675  0.06840139]]

T^2:
223.3101756848916


Our critical point at $\alpha=0.05$ is

$$\frac{(n-1)p}{(n-p)}F_{p,n-p}(\alpha)=\frac{(87-1)3}{(87-3)}F_{3,87-3}(0.05)=\frac{258}{84}F_{3,84}(0.05)$$

In [8]:
print("Critical point:",258/84*stats.f.ppf(1-.05,3,84))

Critical point: 8.3334833261061


We reject our null hypothesis at $\alpha=.05$ because $223.3102>8.3335$.

\pagebreak

# Exercise 5.19

Measurements of $x_1 =$ stiffness and $x_2 =$ bending strength for a sample of n=30 pieces of a particular grade of lumber are given in Table 5.11

## (a) Construct and sketch a 95% confidence ellipse for the pair $[\mu_1,\mu_2]^T$ where $\mu_1=E(X_1)$ and $\mu_2=E(X_2)$

We know that

$$T^2=n(\pmb{\bar{X}}-\pmb{\mu_0})^T \pmb{S^{-1}}(\pmb{\bar{X}}-\pmb{\mu_0})$$

and 

$$T^2\sim\frac{(n-1)p}{(n-p)}F_{p,n-p}$$

So our confidence region is given by 

$$n(\pmb{\bar{X}}-\pmb{\mu_0})^T \pmb{S^{-1}}(\pmb{\bar{X}}-\pmb{\mu_0})\leq\frac{(n-1)p}{(n-p)}F_{p,n-p}(0.05)$$

We are given

$$\pmb{X}=\begin{bmatrix}\pmb{x_1}&\pmb{x_2}\end{bmatrix}$$

First we find 

$$\pmb{\bar{X}}=\begin{bmatrix}\bar{x_1}\\\bar{x_2}\end{bmatrix}$$

Using the mean, we can then compute the sample variance

$$\pmb{S}=\frac{1}{n-1}(\pmb{X-1\bar{x}^T})^T(\pmb{X-1\bar{x}^T})$$

Now we have sufficient information to plug into the equation for our confidence region

$$n(\pmb{\bar{X}}-\pmb{\mu_0})^T \pmb{S^{-1}}(\pmb{\bar{X}}-\pmb{\mu_0})\leq\frac{(n-1)p}{(n-p)}F_{p,n-p}(0.05)$$

$$30\begin{bmatrix}\bar{x_1}-\mu_{1}&\bar{x_2}-\mu_{2}\end{bmatrix} \pmb{S^{-1}}\begin{bmatrix}\bar{x_1}-\mu_{1}\\\bar{x_2}-\mu_{2}\end{bmatrix}\leq\frac{(29)2}{(28)}F_{2,28}(0.05)$$

Additionally, to find the semi-major and semi-minor axis lengths of our confidence ellipse, we use S to compute the eigenvalues and their respective eigenvectors.

$$|\pmb{S}-I\pmb{\lambda}|=0, \text{ and } \lambda_1, \lambda_2 \text{ are our eigenvalues with their respective eigenvectors }\pmb{u_1}, \pmb{u_2}$$

The half-axis lengths from $(\mu_1,\mu_2)$ are given by:

$$\sqrt{\frac{(29)2}{30(28)}F_{2,28}(0.05)\lambda_i}$$ in their respective directions $\pmb{u_1}, \pmb{u_2}$

## (b) Suppose $\mu_{10}=2000, \mu_{20}=10000$ represent "typical" values for the stiffness and bending strength, respectively. Given the result in (a), are the data in Table 5.11 consistent with these values?

We can perform a $T^2$ hypothesis test, we know that

$$T^2=n(\pmb{\bar{X}}-\pmb{\mu_0})^T \pmb{S^{-1}}(\pmb{\bar{X}}-\pmb{\mu_0})=30\begin{bmatrix}\bar{x_1}-2000&\bar{x_2}-10000\end{bmatrix} \pmb{S^{-1}}\begin{bmatrix}\bar{x_1}-2000\\\bar{x_2}-10000\end{bmatrix}$$

And our critical point is given by

$$\frac{(n-1)p}{(n-p)}F_{p,n-p}(\alpha)=\frac{(29)2}{(28)}F_{2,28}(0.05)$$

$H_0:[\mu_{10}=2000, \mu_{20}=10000]^T$

$H_1:[\mu_{10}\neq2000, \mu_{20}\neq10000]^T$

We reject our null hypothesis at $\alpha=.05$ if $T^2>\frac{(29)2}{(28)}F_{2,28}(0.05)$, it is for this same reason that we also fail to reject $H_0$ if the point falls within the confidence ellise.

## (c) Is the bivariate normal distribution a viable population model? Explain the reference to Q-Q plots and a scatter diagram.

A scatter plot is useful in visually determining outliers in our data. We can use Q-Q plots to test for marginal normality of both the stiffness and bending strength. These three diagrams then help us to determine whether bivariate normality is a plausible population model in this scenario. If the Q-Q plots follow linear trends and the scatter plot data coincides, we can then assume it must be plausible.

\pagebreak

# Exercise 5.21

Using the data on bone mineral content in Table 1.8, construct the 95% Bonferroni intervals for the individual means. Also, find the 95% simultaneous $T^2$-intervals. Compare the two sets of intervals.

We have 6 samples $\pmb{X}=[\pmb{x_1},...,\pmb{x_6}]$ and conversely $\pmb{\bar{X}}=[\bar{x_1},...,\bar{x_6}]^T$

We can now compute the sample variance


$$\pmb{S}=\frac{1}{n-1}(\pmb{X-1\bar{x}^T})^T(\pmb{X-1\bar{x}^T})$$

This is a 6x6 matrix from which we will need $s_{11},s_{22},s_{33},s_{44},s_{55},s_{66}$

---

The 95% confidence Bonferroni intervals, given $m=p=6$, for each $\mu_i$ where $i=1,...,6$ is given by

$$\bar{x_i}-t_{n-1}(\frac{\alpha}{2p})\sqrt{\frac{s_{ii}}{n}}\leq\mu_i\leq\bar{x_i}+t_{n-1}(\frac{\alpha}{2p})\sqrt{\frac{s_{ii}}{n}}$$

$$\bar{x_i}-t_{24}(\frac{0.05}{12})\sqrt{\frac{s_{ii}}{25}}\leq\mu_i\leq\bar{x_i}+t_{24}(\frac{0.05}{12})\sqrt{\frac{s_{ii}}{25}}$$

---

The simultaneous 95% confidence $T^2$-intervals are similarly given for each $\mu_i$ where $i=1,...,6$ by

$$\bar{x_i}-\sqrt{\frac{p(n-1)}{n(n-p)}F_{p,n-p}(\alpha)}\sqrt{s_{ii}}\leq\mu_i\leq\bar{x_i}+\sqrt{\frac{p(n-1)}{n(n-p)}F_{p,n-p}(\alpha)}\sqrt{s_{ii}}$$

$$\bar{x_i}-\sqrt{\frac{6(24)}{25(19)}F_{6,19}(0.05)}\sqrt{s_{ii}}\leq\mu_i\leq\bar{x_i}+\sqrt{\frac{6(24)}{25(19)}F_{6,19}(0.05)}\sqrt{s_{ii}}$$

\pagebreak

# Exercise 5.29

Refer to the car body data in Exercise 5.28. These are all measured as deviations from target value so it is appropriate to test the null hypothesis that the mean vector is zero. Using the first 30 cases, test $H_0: \pmb{\mu}=\pmb{0}$ at $\alpha=0.05$


---

To test the first 30 cases using this null hypothesis, we can use a $T^2$ hypothesis test. 

First we need to compute the mean for each of our 6 samples (each using only the first 30 observed values) and the sample variance

$$\pmb{\bar{x}}=\begin{bmatrix}\bar{x_1}\\\bar{x_2}\\\bar{x_3}\\\bar{x_4}\\\bar{x_5}\\\bar{x_6}\end{bmatrix}$$

$$\pmb{S}=\frac{1}{n-1}(\pmb{X-1\bar{x}^T})^T(\pmb{X-1\bar{x}^T})$$

We know the formula for the Hotelling $T^2$, and now have enough information to plug in all the values

$$T^2=n(\pmb{\bar{X}}-\pmb{\mu_0})^T \pmb{S^{-1}}(\pmb{\bar{X}}-\pmb{\mu_0})=50(\pmb{\bar{X}}-\pmb{0})^T \pmb{S^{-1}}(\pmb{\bar{X}}-\pmb{0})=50\begin{bmatrix}\bar{x_1}\\\bar{x_2}\\\bar{x_3}\\\bar{x_4}\\\bar{x_5}\\\bar{x_6}\end{bmatrix}^T \pmb{S^{-1}}\begin{bmatrix}\bar{x_1}\\\bar{x_2}\\\bar{x_3}\\\bar{x_4}\\\bar{x_5}\\\bar{x_6}\end{bmatrix}$$


And our critical point is given by

$$\frac{(n-1)p}{(n-p)}F_{p,n-p}(\alpha)=\frac{(29)6}{(24)}F_{6,24}(0.05)$$

We reject our null hypothesis at $\alpha=.05$ if $T^2>\frac{(29)6}{(24)}F_{6,24}(0.05)$.